va_explorer/va_data_management/tasks.py

```diff
@@ -0,0 +1,13 @@
+from config.celery_app import app
+from va_explorer.va_data_management.utils import coding
+
+
+@app.task()
+def run_coding_algorithms():
```

Comment:
Would it be feasible/helpful to include a `test_tasks.py` in `va_data_management/tests` to cover this task code?